cmd/atlas/internal/docker/docker_test.go

```diff
@@ -105,10 +109,101 @@ func TestFromURL(t *testing.T) {
 	cfg, err = FromURL(u)
 	require.NoError(t, err)
 	require.Equal(t, &Config{
+		driver:   "postgres",
 		Image:    "postgis/postgis:14-3.4",
 		Database: "postgres",
 		Env:      []string{"POSTGRES_PASSWORD=pass"},
 		Port:     "5432",
 		Out:      io.Discard,
 	}, cfg)
 }
+
+func TestFromURL_CustomImage(t *testing.T) {
+	for _, tt := range []struct {
+		url, image, db string
```

Comment:
```suggestion
		url, image, db, dlct string
```

Reply:
Prefer to check for the correct driver chosen as well here. Just a minor suggestion, as the testscript is there as well.